libs/components/modals/src/lib/modules/modal/modal.component.scss

```diff
@@ -70,6 +77,20 @@
     outline-width: thin;
     outline-offset: -1px;
   }
+
+  &:has(> .sky-layout-host-fit) {
+    flex-grow: 1;
+    height: var(--sky-modal-content-max-height, 100%);
+    position: relative;
+  }
+
+  .sky-layout-host-fit ::ng-deep sky-modal-content {
+    padding: 0;
+  }
+
+  .sky-layout-host-none {
```

Comment:
If you eliminate the new `div` with the `skyLayoutHost` directive, this should go away as well.